lib/internal/streams/readable.js

```diff
@@ -1371,6 +1371,18 @@ Readable.from = function(iterable, opts) {
   return from(Readable, iterable, opts);
 };
 
+async function * streamSeriesIterator(streams) {
+  for await (const stream of streams) {
+    for await (const chunk of stream) {
+      yield chunk;
+    }
```

Comment:
Would that have the same effect?
```suggestion
    yield* steam;
```

Reply:
This works fine and looks better. Thanks